libs/utils/src/filenames.ts

```diff
@@ -118,6 +120,30 @@ export function generateCastVoteRecordReportDirectoryName(
   return isTestMode ? `TEST${SECTION_SEPARATOR}${filename}` : filename;
 }
 
+/**
+ * Generates a name for a cast vote record export directory
+ */
+export function generateCastVoteRecordExportDirectoryName({
+  inTestMode,
+  machineId,
+  time = new Date(),
+}: {
+  inTestMode: boolean;
+  machineId: string;
+  time?: Date;
+}): string {
+  const machineString = [
+    'machine',
+    maybeParse(MachineId, machineId) ?? sanitizeString(machineId),
+  ].join(SUBSECTION_SEPARATOR);
+  const timeString = moment(time).format(TIME_FORMAT_STRING);
```

Comment:
As per https://github.com/votingworks/vxsuite/pull/3842#discussion_r1302195906, I'll be relying increasingly on the metadata file instead of the directory name to extract information for display in the UI.

As far as I can tell, the only place that we display the directory name directly is in some error messages, which I'm not too worried about and can easily modify.